src/ontology/mondo-edit.obo

```diff
@@ -31375,7 +31375,7 @@ synonym: "Opitz-Kaveggia syndrome" NARROW [DOID:14711, GARD:0002317]
 xref: DOID:14711 {source="MONDO:equivalentTo"}
 xref: ICD9:759.89 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MESH:C537923 {source="DOID:14711"}
-xref: OMIMPS:305400 {source="MONDO:equivalentTo"}
+xref: OMIMPS:305450 {source="MONDO:equivalentTo"}
```

Comment:
line 31383 should be updated to use this new OMIMPS ID

`relationship: has_characteristic MONDO:0021152 {source="OMIMPS:305400"} ! inherited`

Reply:
Good find! Updated.